features/range.yml

```diff
@@ -0,0 +1,33 @@
+name: Range
```

Comment:
Since we haven't got an answer to whether this should standalone yet, but I also don't want to let this sit idle for forever, here's my idea for how to proceed with this:

Let's move this in to the `draft` folder, add the draft date, and add a comment that links to this PR's discussion. That way, when we do get an answer to whether or not we need some "foundational DOM" feature (or features), then we have at least this part of the work documented somewhere.